types/components/FormFile.d.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import FormFileInput from './FormFileInput';
+import FormFileLabel from './FormFileLabel';
+import { BsPrefixComponent } from './helpers';
+
+export interface FormFileProps {
+  bsCustomPrefix?: string;
+  innerRef?: React.LegacyRef<this>;
```

Comment:
We don't have a propType for this and this component is already forwarding its ref properly, and we probably don't need to support it so we can just remove this.

Reply:
Probably no reason at all. Left from FormCheck that I used as a basis for this. Removing since refs seems to be working without it.